test/assets/unrunnableTestCases.csv

```diff
@@ -2096,6 +2088,7 @@ K2-Axes-49,Error: No selector counterpart for: computedDocumentConstructor.
+K2-Axes-88,AssertionError: Expected XPath declare variable $i := <e> <a>1</a> <b>2</b> </e>; <result> { ($i/b, $i/a)/. } </result> to resolve to the given XML. Expected <result><b>2</b><a>1</a></result> to equal <result><a>1</a><b>2</b></result>
```

Comment:
#[Same as in this PR](https://github.com/FontoXML/fontoxpath/pull/159#discussion_r318608602)

Reply:
This is weird. Just checked the result of the test on both `master` and this branch, and the result is the same. The expected result of the test is not correct.